README.md

```diff
@@ -97,11 +97,11 @@ $ eksctl create cluster
 $
 ```
 
-Install `eksctl` following the [installation instructions](https://eksctl.io/introduction/installation).
+Install `eksctl` following the [installation instructions](https://eksctl.io/intro/#installation).
```

Comment:
these failures are due to recent rollout of new website

Reply:
We should move back to the original URL - since this might break external ilnks. Do you think you can open a new PR for this?

cc @martina-if @cPu1

Reply:
I went ahead and did in https://github.com/weaveworks/eksctl/pull/2102

Reply:
Thanks for spotting this @sayboras !

Reply:
:100: